tests/TransactionStackTest.php

```diff
@@ -30,7 +30,9 @@ public function testSimple()
         $this->assertEquals($stack->peek(), null);
 
         $stack->clear();
-        $this->assertInternalType('array', $stack->stack);
-        $this->assertEquals(0, count($stack->stack));
+        $stack_r = new \ReflectionProperty($stack, 'stack');
```

Comment:
PHPUnit provides an `assertAttributeEquals` method to check whether an attribute (even private) equals a given value